src/Plugin/Block/ActivityFinder4Block.php

```diff
@@ -359,18 +360,34 @@ public function blockForm($form, FormStateInterface $form_state) {
     $form['additional']['start_month_filter'] = [
       '#type' => 'checkbox',
       '#title' => $this->t('Start month'),
+      '#description' => $this->t('Allow users to filter by start month. This option has no additional configuration.'),
       '#default_value' => $conf['start_month_filter'],
     ];
 
     $form['additional']['in_memberships_filter'] = [
       '#type' => 'checkbox',
       '#title' => $this->t('In Membership'),
+      '#description' => $this->t('Allow users to filter by sessions that are included in their membership. This filters on the ‘In membership’ field on Sessions.'),
       '#default_value' => $conf['in_memberships_filter'],
     ];
 
     $form['additional']['duration_filter'] = [
       '#type' => 'checkbox',
       '#title' => $this->t('Duration'),
+      '#description' => $this->t('Allow users to search by the length of the session. Durations are configurable in the @link.', [
+        '@link' => Link::createFromRoute(
+          'Activity Finder Settings',
+          'openy_activity_finder.settings',
+          [],
+          [
+            'attributes' => [
+              'title' => 'Activity Finder Settings',
```

Comment:
```suggestion
```
Setting a title that's the same as the link text is unnecessary.

Reply:
@froboy thanks! Fixed.